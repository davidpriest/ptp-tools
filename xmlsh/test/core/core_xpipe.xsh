# Test of xpipes 

# first test string pipes

xecho <[ 1,"hello",<elem/>,document { <doc/> } , attribute attr {"b"} ]> | 
while read a ; do xtype $a ; done 

# Now with xpipes 
set -xpipe
xecho  <[ 1,"hello",<elem/>,document { <doc/> } , attribute attr {"b"} ]> | 
while xread a ; do xtype $a ; done 
