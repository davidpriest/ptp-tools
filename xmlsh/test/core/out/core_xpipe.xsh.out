xs:string
xs:string
xs:string
xs:string
xs:string
xs:integer
xs:string
element()
document-node()
attribute()
